docs/common/migrations.md

```diff
@@ -29,9 +29,11 @@ These SQL statements are run by the `Database.Schema.migrate()` method. Migratio
 
 ## Verifying Migrations
 
-You can also place a `.db` file in the `src/main/sqldelight` folder of the same `<version number>.db` format. If there is a `.db` file present, a new `verifySqlDelightMigration` task will be added to the gradle project, and it will run as part of the `check` task, meaning your migrations will be verified against that `.db` file. It confirms that the migrations yield a database with the latest schema.
+A `verifySqlDelightMigration` task will be added to the gradle project, and it will run as part of the `check` task. For any `.db` file named `<version number>.db` in your SqlDelight source set (e.g. `src/main/sqldelight`) it will apply all migrations starting from `<version number>.sqm`. and confirms that the migrations yield a database with the latest schema.
```

Comment:
Migrations are run on all `.db` files? If no, how is it decided which sqm runs with with db?

Reply:
Yes, I believe they are run on every `.db` file that is present.

So for the following files:

```
1.db
2.db
3.db
1.sqm
2.sqm
3.sqm
```

The following will get run:

```
1.sqm 2.sqm 3.sqm
2.sqm 3.sqm
3.sqm
```

Reply:
Yup, that's why you only need 1 db file.

Reply:
hm. im a bit confused. My current project has like 5.db files because I thought that's what was required in order to test each db migration path.

Reply:
I had over 100 .db files before I realized this. Verifying migrations took forever 😵‍💫

Reply:
so... i dont need a separate db file for each migration. hm. i understand how that helps speed. but wouldn't that mean that im not verifying each possible permutation of a migration?

Reply:
You'll have every one covered.

```
1.db
1.sqm
2.sqm
3.sqm

1.db -> apply 1.sqm -> apply 2.sqm -> apply 3.sqm -> verify that the db matches your schema
```
vs

```
1.db
2.db
3.db
4.db
1.sqm
2.sqm
3.sqm

1.db -> apply 1.sqm -> apply 2.sqm -> apply 3.sqm -> verify that the db matches your schema
2.db -> apply 2.sqm -> apply 3.sqm -> verify that the db matches your schema
3.db -> apply 3.sqm -> verify that the db matches your schema
4.db -> verify that the db matches your schema
```